.git-blame-ignore-revs

```diff
@@ -1,3 +1,6 @@
 # Conversion to new code style
 45b9887603f08523f9a663b164bd9f10b85f8b25
 9114d609168b327f00ef47b9891733cdb119b512
+
+#License update
+b6b6d463c0b5452014f46e088a50ec3aca5e5392
```

Comment:
It is not necessary or desirable to exclude the license update from the `git blame` output. The license update does not affect blame output for any lines of actual code, and excluding it simply makes Git work harder to [find a previous commit](https://www.git-scm.com/docs/git-blame#Documentation/git-blame.txt---ignore-revltrevgt) where a nearby line was changed.